dotcom-rendering/cypress/e2e/parallel-3/article.interactivity.cy.js

```diff
@@ -15,8 +16,12 @@ describe('Interactivity', function () {
 	});
 
 	describe('Verify elements have been hydrated', function () {
-		it('should open the edition dropdown menu when clicked and hide when expected', function () {
+		// eslint-disable-next-line mocha/no-exclusive-tests
+		it.only('should open the edition dropdown menu when clicked and hide when expected', function () {
 			cy.visit(`/Article?url=${articleUrl}`);
+			cy.percySnapshot('Article', {
+				widths: [320, 375, 480, 660, 740, 980, 1140, 1300],
```

Comment:
Nit: You can get these by importing `breakpoints` from Source

Reply:
Yep it was just something rough for demo purposes :)